src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &__product-image {
+    padding: 16px 3px 0;
+    display: flex;
+    width: 160px;
+    height: 134px;
```

Comment:
Consider using variables for width and height values that are reused. This will make your code more maintainable and easier to read.